src/Forms/PresentedOptionsField.php

```diff
@@ -99,4 +111,20 @@ public function setResource(Resource $resource)
         $this->resource = $resource;
         return $this;
     }
+
+    /**
+     * @return array
+     */
+    public function getSelectedFields()
+    {
+        return $this->selectedFields;
+    }
+
+    /**
+     * @param array $selectedFields
+     */
+    public function setSelectedFields($selectedFields)
```

Comment:
Type hint array?

Reply:
In the end this wasn't required. I'll just remove it.